test/FixtureFactory.php

```diff
@@ -25,12 +25,17 @@ public static function createPorter()
 
     public static function createCard()
     {
-        return new Card('4242424242424242', 12, date('Y') + 1, '123');
+        return static::createCardFromNumber('4242424242424242');
     }
 
     public static function createInvalidCard()
     {
-        return new Card('0', 12, date('Y') + 1, '123');
+        return static::createCardFromNumber('0');
```

Comment:
Since this class is marked `final` there is no benefit from using LSB. Prefer `self` here and throughout this file.